x/blockdb/database.go

```diff
@@ -176,6 +177,7 @@ type Database struct {
 	log        logging.Logger
 	closed     bool
 	fileCache  *lru.Cache[int, *os.File]
+	entryCache *lru.Cache[BlockHeight, BlockData]
```

Comment:
entry cache is a really poor name to pick for a cache that caches blocks.

Reply:
I have plans to rename this package to `heightindexdb` since the db doesn't need to store just blocks. I'm planning to do this before moving the package out of `x`.
Given that, I didn't opt to include `block` in the name. I've also considered `entryDataCache` instead of `entryCache`, thoughts?

Reply:
I'll name it to `blockCache` now. We can rename in a single PR if needed when we move the package to `database.heightindexdb`